crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -1007,6 +1008,17 @@ impl RoomInfo {
         }));
     }
 
+    /// Update the room avatar
+    pub fn update_avatar(&mut self, url: Option<OwnedMxcUri>) {
+        let mut content = RoomAvatarEventContent::new();
+        content.url = url;
+
+        self.base_info.avatar = Some(MinimalStateEvent::Original(OriginalMinimalStateEvent {
```

Comment:
Should we set this to `None` when the url is `None`, instead of having an event with empty data?

Reply:
🤦 of course yeah…